app/css/bootstrap.css

```diff
@@ -5934,6 +5934,8 @@ button.close {
 .modal-header {
   padding: 15px;
   border-bottom: 1px solid #e5e5e5;
+  display: flex;
+  justify-content: flex-end;
```

Comment:
changing *all* modal headers across the app - surely this will affect all bootstrap modals and we don't want that? does it affect other modals?